src/format/opamVariable.mli

```diff
@@ -36,18 +36,28 @@ module Full: sig
 
   include OpamStd.ABSTRACT
 
-  (** Create a variable local for a given library/syntax extension *)
-  val create: OpamPackage.Name.t -> variable -> t
+  type scope =
+    | Global (** Note: this is attributed to unqualified variables, and may
+                 also design self-referring ones *)
+    | Self (** Variable in a package-specific file referring to that
+               package [-:varname] *)
```

Comment:
Indeed (attempted `-:` but that didn't go well with the parser. And we don't want to introduce files that don't parse with older versions...)